src/app/modules/attribution/view-maprule.html

```diff
@@ -26,10 +26,10 @@ <h5>Geometry</h5>
 								<ng-container *ngFor="let geometry of preset.geometry">
 									<div class="view-panel-content view-panel-pill-group d-inline-flex">
 										<div [ngSwitch]="geometry" class="view-panel-pill-geometry">
-											<app-area *ngSwitchCase="'area'"></app-area>
-											<app-node *ngSwitchCase="'node'"></app-node>
-											<app-way *ngSwitchCase="'way'"></app-way>
-											<app-area *ngSwitchCase="'closedway'"></app-area>
+											<app-area *ngSwitchCase="'Area'"></app-area>
+											<app-node *ngSwitchCase="'Point'"></app-node>
+											<app-way *ngSwitchCase="'Line'"></app-way>
+											<app-area *ngSwitchCase="'Line'"></app-area>
```

Comment:
I think we can remove the second line element as it might be causing some double icon action
![duplicate-geometries](https://user-images.githubusercontent.com/15100429/49184310-ffc4d380-f32c-11e8-965d-ebb729bceace.gif)